docs/src/plotting_utils.jl

```diff
@@ -192,29 +191,31 @@ function verify_convergence(
         plot_kwargs...,
     )
 
-    scb_cur_avg_err = make_saving_callback(cur_avg_err, prob.u0, t_end, nothing)
-    scb_cur_avg_sol_and_err = make_saving_callback(cur_avg_sol_and_err, prob.u0, t_end, nothing)
-
     for algorithm_name in algorithm_names
+        @info algorithm_name
         alg = algorithm(algorithm_name)
         alg_str = string(nameof(typeof(algorithm_name)))
         predicted_order = predicted_convergence_order(algorithm_name, prob.f)
         linestyle = linestyles[(predicted_order - 1) % length(linestyles) + 1]
 
         verbose && @info "Running $test_name with $alg_str..."
         plot1_net_avg_errs = map(plot1_dts) do plot1_dt
+            # @show alg.tableau.imp
             cur_avg_errs =
                 solve(
                     deepcopy(prob),
                     alg;
                     dt = plot1_dt,
                     save_everystep = !only_endpoints,
-                    callback = scb_cur_avg_err,
+                    save_func = cur_avg_err,
+                    kwargshandle = DiffEqBase.KeywordArgSilent,
```

Comment:
I think this may have been an incorrect merge resolution: `save_func` is not documented anywhere (it seems to be internal), so we shouldn't use it. Also, I don't like the idea of using `DiffEqBase.KeywordArgSilent`.

Reply:
@dennisYatunin

Reply:
They were failing before this change. They've been periodically breaking on main for months. So, it's not clear to me that this needs to be reverted.

Reply:
In fact, I don't believe that these changes should change the results, so I'd be surprised if this was responsible for the docs breaking.

Reply:
It would also be good to add `@test` or `@assert` calls to the convergence tests to avoid having them silently break for months.

Reply:
The problem is that it's flakey, it passes locally, or on push, and then fails on the main branch.

Another issue is that virtually nothing is printed when it errors, it's not clear which example is breaking, and what any of the types are. I think a better solution would be to run these examples on buildkite, and have a dependent job make a summary.

I don't want to add the use of internals back into this repo, so please fix the issue without `save_func /KeywordArgSilent`, or fix the underlying issue, or leave it broken, as it has been (and very well may still be in this repo as the issue is not easily reproducible).